sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client_operations.hpp

```diff
@@ -0,0 +1,100 @@
+﻿
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @file
+ * @brief Defines the Key Vault Certificate llong running operations.
+ *
+ */
+
+#pragma once
+
+#include <azure/core/internal/client_options.hpp>
+
+#include "azure/keyvault/certificates/certificate_client_models.hpp"
+#include "azure/keyvault/certificates/dll_import_export.hpp"
+
+#include <azure/core/http/http.hpp>
+#include <azure/core/operation.hpp>
+#include <azure/core/operation_status.hpp>
+#include <azure/core/response.hpp>
+
+namespace Azure { namespace Security { namespace KeyVault { namespace Certificates {
+  class CertificateClient;
+  /**
+   * @brief Represents a create certificate long running operation
+   */
+  class CreateCertificateOperation final : public Azure::Core::Operation<KeyVaultCertificate> {
+
+  private:
+    friend class CertificateClient;
```

Comment:
nit: up to you, but I think it makes sense to declare friends the very first line before any visibility level declaration, i.e. before `private:`.